build.sbt

```diff
@@ -55,6 +58,20 @@ lazy val root = project
     parallelExecution in Test in ThisBuild := false
   )
 
+lazy val cats = crossProject
+  .configureCross(moduleCrossConfig("cats"))
+  .dependsOn(core % "compile->compile;test->test")
+  .settings(
+    libraryDependencies += "org.typelevel" %%% "cats-core" % catsVersion,
+    initialCommands += s"""
+      import $rootPkg.cats._
+      import $rootPkg.cats.auto._
```

Comment:
There is no `cats.auto` object in this PR. Running `cats/console` will fail with an error, right?